CommunityToolkit.Common/Extensions/ISettingsStorageHelperExtensions.cs

```diff
@@ -3,7 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections.Generic;
-using CommunityToolkit.Helpers;
+using CommunityToolkit.Common.Helpers;
 
 namespace CommunityToolkit.Common.Extensions;
```

Comment:
There was also the question of here if the `Extensions` should be removed here as no other extensions use this namespace, eh?

Reply:
Should this be moved to `Helpers` then?